libs/ledger-live-common/src/families/evm/api/rpc.common.ts

```diff
@@ -13,8 +13,9 @@ import { getSerializedTransaction } from "../transaction";
 import { makeLRUCache } from "../../../cache";
 import ERC20Abi from "../abis/erc20.abi.json";
 
-export const RPC_TIMEOUT = 5000; // wait 5 sec after a fail
-export const DEFAULT_RETRIES_RPC_METHODS = 3;
+export const RPC_TIMEOUT = process.env.NODE_ENV === "test" ? 100 : 5000; // wait 5 sec after a fail
```

Comment:
No it's actually changed only for the "integration tests" to either succeed faster or fail faster. With the 5s timeout, for a human there is no problem except "it's a bit long" (but necessary for some nodes like Optimism that have a very strict ratelimit), but for the bridge integration test which is trying to compare 2 "prepared" transactions, the 5s timeout can change completely the fees estimations because the network will have changed. So end result is transaction1 & transaction2 are different because of the network, but not because of our implementation, so it triggers a false negative 🤷‍♂️ 

So in fine, this timeout in "test" mode is just here to retry if it fails and wait the bare minimum amount of time, and if it fails then it's still fine as integration tests are flaky and optional (for now at least).